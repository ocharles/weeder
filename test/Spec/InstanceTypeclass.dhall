{ roots = [ "Spec.InstanceTypeclass.InstanceTypeclass.\\$fRootClassChar" ] : List Text, type-class-roots = False }
