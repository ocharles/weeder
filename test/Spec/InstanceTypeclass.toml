roots = []

type-class-roots = false

root-instances = [ "RootClass Char" ]
