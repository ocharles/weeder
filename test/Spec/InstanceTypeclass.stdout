main: test/Spec/InstanceTypeclass/InstanceTypeclass.hs:4:1: Foo
main: test/Spec/InstanceTypeclass/InstanceTypeclass.hs:10:1: (Instance) :: Foo Char
